doc/source/user_guide/io.rst

```diff
@@ -1565,7 +1565,7 @@ of header key value mappings to the ``storage_options`` keyword argument as show
 
 All URLs which are not local files or HTTP(s) are handled by
 `fsspec`_, if installed, and its various filesystem implementations
-(including Amazon S3, Google Cloud, SSH, FTP, webHDFS...).
+(including Amazon S3, Google Cloud, Hugging Face, SSH, FTP, webHDFS...).
```

Comment:
This is meant to give examples and not be an exhaustive list. If we leak ecosystem entries into various parts of the docs, it seems to me we are at great risk of leaving behind stale docs as the ecosystem evolves over time. I'd prefer to remove this.

Reply:
I see, I reverted the change. Let me know if there is a better place to list the main fsspec implementations